android/README.md

```diff
@@ -0,0 +1,59 @@
+﻿# General
+
+We follow the [Google Java Code Style](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html), with some exceptions:
+
+**3.3.1** Wildcard imports are allowed in unit tests
+
+**4.1.1** You can ommit braces for single-statement `if` (but not for single-statement `for`).
```

Comment:
I'm not sure if it is a valid point, and it doesn't really affect me, but is this a smart thing to do? I can't see how it increases readability or effectivity, and I think keeping the coding style as similar as possible across platform and code bases is a good thing.

Thoughts?